content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/deploy-corda-cluster.md

```diff
@@ -0,0 +1,388 @@
+---
+date: '2021-11-14'
+title: "Deploying a Corda Cluster"
+menu:
+  corda-5-alpha:
+    parent: corda-5-alpha-tutorials-deploy
+    identifier: corda-5-alpha-tutorial-deploy-k8s
+    weight: 1000
+section_menu: corda-5-alpha
+---
+
+This page describes how to deploy Corda 5 Alpha. It assumes all necessary [prerequisites](../prerequisites.html) have been installed.
+
+## Download and Push Docker Images to a Registry
+
+The Corda Docker images must be in a Docker registry that is accessible from the Kubernetes cluster in which Corda will run. The images are provided in a `tar` file which can be loaded into a local Docker engine and then pushed from there to the registry.
+
+1. Download `corda-worker-images-Eagle.tar` from the [R3 Customer Hub](https://r3.force.com/).
+
+2. Inflate and load the `corda-worker-images-Eagle.tar` file into the local Docker engine with the following command:
+   ```shell
+   docker load -i corda-worker-images-Eagle.tar
+   ```
+
+3. Retag each image using the name of the registry to be used and push the image. The following is an example script to automate this. It takes the target Docker registry as an argument.
+   ```shell
+   #!/bin/bash
+   if [ -z "$1" ]; then
+    echo "Specify target registry"
+    exit 1
+   fi
+
+   declare -a images=(
+    "corda-os-rpc-worker" "corda-os-flow-worker"
+    "corda-os-member-worker" "corda-os-p2p-gateway-worker"
+    "corda-os-p2p-link-manager-worker" "corda-os-db-worker"
+    "corda-os-crypto-worker" "corda-os-plugins" )
+   tag=5.0.0.0-Eagle
+   target_registry=$1
+
+   for image in "${images[@]}"; do
+    source=corda/$image:$tag
+    target=$target_registry/$image:$tag
+    echo "Publishing image $source to $target"
+    docker tag $source $target
+    docker push $target
+   done
+
+   docker tag postgres:14.4 $target_registry/postgres:14.4
+   docker push $target_registry/postgres:14.4
+   ```
+
+## Download Helm Charts
+
+1. Download the Helm charts `corda-5.0.0-Eagle.tgztar` file from the [R3 Customer Hub](https://r3.force.com/).
```

Comment:
```suggestion
* Download the Helm charts `corda-5.0.0-Eagle.tgztar` file from the [R3 Customer Hub](https://r3.force.com/).
```
I suggest using ul instead of ol when there's only one step.